src/schema/schema_create.c

```diff
@@ -433,6 +433,57 @@ __wt_find_import_metadata(WT_SESSION_IMPL *session, const char *uri, const char
     return (0);
 }
 
+/*
+ * __schema_is_tiered_storage_shared --
+ *     Check whether the table is configured for a tiered storage shared.
+ */
+static void
```

Comment:
Instead of passing in a boolean pointer, why not just have the function return true or false?

Reply:
Thanks, fixed.